Firebase/Utilities/Environment/third_party/GULAppEnvironmentUtil.m

```diff
@@ -125,6 +125,24 @@ static BOOL isAppEncrypted() {
   return NO;
 }
 
+static BOOL hasSCInfoFolder() {
```

Comment:
Free functions should start with a capital letter. Here and below.

http://google.github.io/styleguide/objcguide.html#function-names